package.json

```diff
@@ -30,6 +30,7 @@
   },
   "devDependencies": {
     "chai": "*",
+    "hubot-mock-adapter": "*",
```

Comment:
Since this depends on a released version of Hubot, we wouldn't catch any breaking API changes. Maybe we need to use something like [mockery](https://github.com/mfncooper/mockery), to force [their `require('hubot/src/adapter')`](https://github.com/blalor/hubot-mock-adapter/blob/5cb366c9150693624050bc2b04e39c83470de5e8/index.js#L23) to be relative? Does that make sense?